addons/web/static/src/js/tour.js

```diff
@@ -21,7 +21,16 @@ if (typeof openerp.Tour !== "undefined") {
 }
 
 /////////////////////////////////////////////////
-
+/* To check if element is on screen or not */
+$.fn.isOnScreen = function(){
+    var viewport = {};
+    viewport.top = $(window).scrollTop();
+    viewport.bottom = viewport.top + $(window).height();
+    var bounds = {};
+    bounds.top = this.offset().top;
+    bounds.bottom = bounds.top + this.outerHeight();
+    return ((bounds.top <= viewport.bottom) && (bounds.bottom >= viewport.top));
+};
```

Comment:
Its looks like work around solution, instead of this plugin better to use some library.
like this https://github.com/moagrius/isOnScreen/blob/master/jquery.isonscreen.js .
Same for comment for version 8 branch.